simson/common/data_transformations.py

```diff
@@ -67,28 +67,21 @@ def extrapolate_stock(
 
 def scale_parameter_to_future(historic_parameter : NamedDimArray, scaler : NamedDimArray) -> NamedDimArray:
     """Scale parameter to future."""
-    dims = historic_parameter.dims.union_with(scaler.dims)
-
-    scaling_year = historic_parameter.dims.dim_list[0].items[-1]
     n_historic = historic_parameter.shape[0]
+    dims = historic_parameter.dims.union_with(scaler.dims)
 
-    transform_t_thist = get_subset_transformer(dims=dims, dim_letters=('t', 'h'))
     future_parameter = get_new_array(dims, dim_letters=('t',) + historic_parameter.dims.letters[1:])
-    historic_scaler = get_new_array(dims=dims, dim_letters=('h',)+scaler.dims.letters[1:])
-    historic_share = get_new_array(dims=dims, dim_letters=historic_parameter.dims.letters)
-
-    historic_scaler[...] = scaler * transform_t_thist
-    historic_scaler[...] =  historic_scaler.maximum(1e-10)  # have a minimum value to avoid division by zero
-
-    historic_share[...] = historic_parameter / historic_scaler
+    future_parameter.values[:n_historic] = historic_parameter.values
```

Comment:
We don't need this line, right? We have it later anyway...

Reply:
Yeah, you're right, Jakob changed the structure now anyways though so I guess it's resolved :)